Gemfile

```diff
@@ -65,21 +67,26 @@ gem 'rollbar'
 gem 'pghero'
 
 # SEO Meta Tags
-gem 'metamagic'
 gem 'meta-tags'
+gem 'metamagic'
 
 gem 'newrelic_rpm'
 
 gem 'recipient_interceptor'
 
+# Solves 'NoMethodError: assert_template has been extracted to a gem.' as suggested by rspec
+# This error was thrown when using `expect(response).to render_template('template')`
+gem 'rails-controller-testing'
```

Comment:
Why do we need this gem? If it is used for testing, add it under the :test group